### PR TITLE
Binary Search in c++, cc @MJ10

### PR DESCRIPTION
**Pull request by krahulreddy**
This Pull Request resolves Issue #25 

Description:
In this Pull Request I have written a c++ program to perform binary search on a sorted array.

### CODE REVIEW

File: Binary Search/krahulreddy.cpp [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+int main(){
+int a[100], n, ele, low=0, mid, high, pos=-1;
+cout<<"Enter the number of elements : ";
+cin>>n;
+cout<<"Enter the elements : ";
+for(int i=0;i<n;i++)
+cin>>a[i];
+cout<<"Enter Search element : ";
+cin>>ele;
```
Comment (utkarsh23) [inline Binary Search/krahulreddy.cpp]: Would be nicer to let the user know which element number is being input each time a value is asked for. Along with your input statement, add line 8 also into the for loop and modify it to display the element number that is being input. Also try to reindent your code.

**Comment by salman-bhai** [comment]
Are there any updates or should we close this PR as `spam`?

File: Binary Search/krahulreddy.cpp (hunk 1 above)
Comment (krahulreddy) [reply Binary Search/krahulreddy.cpp]: I'll change it soon...

**Comment by salman-bhai** [comment]
PR closed due to inactivity

**Comment by salman-bhai** [comment]
> merge conflicts. Please resolve them.

**Pull request closed without merging**